### ISSUE TITLE
creating a new collected material should trigger an “H” in the todo list

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#34]
a little help for the registrar. Also:
- making it a small find should trigger a “R” in the todo list 
- creating a cm with material ceramics should trigger a “C” in the todo list

**Comment by urapadmin** [comment]
since registration codes are configurable this either needs a code-independent implementation or should be limited to urap.
Not necessary for v1

**Comment by urapadmin** [comment]
see #1817

**Comment by urapadmin** [comment]
this should be covered by #1817. It just need to be configured for projects the way this ticket originally wanted.
So I leave this for URAP

**Comment by luizaogs** [comment]
@lbestock I leave this one for you since I don't think I should be configuring URAP.

**Comment by lbestock** [comment]
I wanted to clear this one ticket from testing but even that takes more brain space than I have, so I must not touch it tonight.

**Comment by lbestock** [comment]
Better mentioned here than in #1813 where I first recorded it; this is not working properly when I attempt to actually configure URAP.

Configuring URAP test.
I did this on my mac after uploading my patch and feeling appreciative of Kiosk, which then let me see the new configuration key on my laptop.

The current registration_code_cm_type_defaults is:
bulk=H small_find_H, P, R sample=H, R
This is the expected actual configuration for URAP.
But making a small find has not triggered the application of any registration todo codes that I can see in the CM record, nor does making a small find. The codes are properly applied for bulk, and the material code for ceramic is properly applied once ceramic is chosen.

**Comment by luizaogs** [comment]
@lbestock : I just tried these configurations in PVD because I wanted to make sure I hadn't messed up yesterday and they worked just fine. My registration_code_cm_type_defaults looks like this: 

<img width="401" alt="Screenshot 2022-11-22 at 9 27 38 AM" src="https://user-images.githubusercontent.com/67970402/203353787-68315b25-44dc-4a80-9285-98ead02195cd.png">

You have some spaces in yours and no = after small_find so I wonder if that might be why yours isn't working.

**Comment by lbestock** [comment]
I did have a = but you were right about the spaces. I wish I had a clearer head.

However, taking care of it did not change the behavior. Here is what my configuration looks like:
![E05E322F-C06A-4A9F-A4AA-E04C9DF99C20](https://user-images.githubusercontent.com/49291909/203391064-f2757c84-056f-4836-b1ee-973c2df532f0.jpeg)

Creating a new small find or a new sample does not result in the application of any to do codes; creating a new bulk does give me the H.

**Comment by urapadmin** [comment]
the issue stems from #889 
As I suspected, valuelists cannot be properly managed there. This is the proof. After I deleted all the line breaks and made new line breaks for the configuration key everything worked fine. Laurel used this particularly key to test #889. The values look the same but they are not. Filemaker expects a different line-break character than the browser to signal a new line. I will look into the matter in #889. But here the safest way is to delete the contents of the field and type it in again (don't copy / paste!) in filemaker.
